game/tests.py

```
@@ -107,6 +108,40 @@ def test_unknown_url_renders_themed_404_page(self):
         self.assertContains(response, reverse('landing'), status_code=404)
 
 
+class SecuritySettingsConfigTest(SimpleTestCase):
+    """Security transport defaults should be explicit and testable."""
+
+    def test_env_bool_uses_default_when_variable_missing(self):
+        with mock.patch.dict(project_settings.os.environ, {}, clear=True):
+            self.assertTrue(project_settings.env_bool('MISSING', True))
```

Ruff (0.15.14)

[warning] Boolean positional value in function call

(FBT003)